spec/system/admin/users_system_spec.rb

```diff
@@ -22,13 +23,18 @@
       click_link "Edit", match: :first
       expect(page).to have_content("Update #{@organization_admin.name}")
       fill_in "user_name", with: "TestUser"
-      fill_in "user_password", with: "123password!"
-      fill_in "user_password_confirmation", with: "123password!"
       click_on "Save"
 
       expect(page.find(".alert")).to have_content "TestUser updated"
     end
 
+    it "cannot edit a user's password" do
+      visit admin_users_path
+      click_link "Edit", match: :first
+      expect(page).not_to have_field("user_password")
+      expect(page).not_to have_field("user_password_confirmation")
+    end
```

Comment:
I don't think there's a need to add a test validating that a feature _doesn't_ exist :)